CMakeLists.txt

```diff
@@ -717,6 +720,12 @@ if(ENABLE_WX)
   endif()
 endif()
 
+if(USE_DISCORD_PRESENCE)
+  message(STATUS "Using static DiscordRPC from Externals")
+  add_definitions(-DUSE_DISCORD_PRESENCE)
```

Comment:
This should likely be added to the targets where this is applicable directly with [target_compile_definitions](https://cmake.org/cmake/help/latest/command/target_compile_definitions.html?highlight=target_#command:target_compile_definitions) instead of add_definitions, otherwise this adds the definition to every single library or executable target within this directory and below (including all core-related stuff).